client/src/components/PodTable.tsx

```diff
@@ -31,51 +35,46 @@ const useStyles = makeStyles(() =>
   })
 )
 
-function calculateRecordPerUser(
+export function PodTable(props: {
   pod: Pod
-): { participantId: number; wins: number; losses: number }[] {
-  return pod.participants.map(participant => {
-    const matchesForParticipant = pod.matches.filter(
-      match => match.playerAId === participant.id || match.playerBId === participant.id
-    )
-    const matchesWon = matchesForParticipant.filter(match => match.winnerId === participant.id)
```

Comment:
We could reuse this code and just add the dropped matches to the filter statement.
So:
const matchesWon = matchesForParticipant.filter(match => !participant.dropped && match.winnerId === participant.id )
And: 
const matchesLost = matchesForParticipant.filter(
      match => participant.dropped || match.winnerId && match.winnerId !== participant.id 
)

We could then use the record object to display the record string, as before